app/services/ApiService.scala

```diff
@@ -41,7 +42,7 @@ class ApiService @Inject() (
       .flatMap {
         _.departures.find(_.localReferenceNumber == lrn).traverse {
           departure =>
-            apiConnector.getMessageTypesByPath(departure.path).map {
+            apiConnector.getMessageTypesByPath(departure.path)(ec, hc, Implicits.readFromJson).map {
```

Comment:
I'm surprised these need defining explicitly... Does the import just need changing to `import uk.gov.hmrc.http.HttpReads.Implicits._` ?